charmcraft.yaml

```diff
@@ -49,7 +49,7 @@ requires:
   database:
     interface: postgresql_client
     limit: 1
-  log-proxy:
+  logging:
```

Comment:
Recommended name of the relation is `logging` (see [Logging via Pebble Log Forwarding](https://discourse.charmhub.io/t/loki-k8s-docs-logging-via-pebble-log-forwarding/15263)).